src/Tool/appsettings.json

```diff
@@ -6,26 +6,25 @@
     },
     "AllowedHosts": "*",
   "DefaultPackageVersions": [
-    "Microsoft.Quantum.Compiler::0.12.20062505-beta",
+    "Microsoft.Quantum.Compiler::0.11.2006.2203-beta",
 
-    "Microsoft.Quantum.CsharpGeneration::0.12.20062505-beta",
-    "Microsoft.Quantum.Development.Kit::0.12.20062505-beta",
-    "Microsoft.Quantum.Simulators::0.12.20062505-beta",
-    "Microsoft.Quantum.Xunit::0.12.20062505-beta",
+    "Microsoft.Quantum.CsharpGeneration::0.11.2006.2203-beta",
+    "Microsoft.Quantum.Development.Kit::0.11.2006.2203-beta",
+    "Microsoft.Quantum.Simulators::0.11.2006.2203-beta",
+    "Microsoft.Quantum.Xunit::0.11.2006.2203-beta",
 
-    "Microsoft.Quantum.Standard::0.12.20062505-beta",
-    "Microsoft.Quantum.Chemistry::0.12.20062505-beta",
-    "Microsoft.Quantum.Chemistry.Jupyter::0.12.20062505-beta",
-    "Microsoft.Quantum.MachineLearning::0.12.20062505-beta",
-    "Microsoft.Quantum.Numerics::0.12.20062505-beta",
+    "Microsoft.Quantum.Standard::0.11.2006.2203-beta",
+    "Microsoft.Quantum.Chemistry::0.11.2006.2203-beta",
+    "Microsoft.Quantum.Chemistry.Jupyter::0.11.2006.2203-beta",
+    "Microsoft.Quantum.MachineLearning::0.11.2006.2203-beta",
+    "Microsoft.Quantum.Numerics::0.11.2006.2203-beta",
 
-    "Microsoft.Quantum.Katas::0.12.20062505-beta",
+    "Microsoft.Quantum.Katas::0.11.2006.2203-beta",
 
-    "Microsoft.Quantum.Research::0.12.20062505-beta",
-    
+    "Microsoft.Quantum.Research::0.11.2006.2203-beta",
 
-    "Microsoft.Quantum.Providers.IonQ::0.12.20062505-beta",
-    "Microsoft.Quantum.Providers.Honeywell::0.12.20062505-beta",
-    "Microsoft.Quantum.Providers.QCI::0.12.20062505-beta"
+    "Microsoft.Quantum.Providers.IonQ::0.11.2006.2203-beta",
+    "Microsoft.Quantum.Providers.Honeywell::0.11.2006.2203-beta",
+    "Microsoft.Quantum.Providers.QCI::0.11.2006.2203-beta"
```

Comment:
Same comment as in Core.csproj, probably better to keep the newer versions here. (The CI build may fail anyway due to this.)

Reply:
Yes, sorry, I was sort of expecting this to fail but wanted to try it anyway. I'll rebuild the packages.

Reply:
Surprisingly, it didn't fail.

Reply:
Interesting, I guess this one was new enough then.